controller/templates/statefulset.yaml

```
@@ -177,4 +179,26 @@ spec:
             limits:
               cpu: 100m
               memory: 32Mi
+        {% else %}
+        - name: passthrough-proxy
+          image: traefik:v2.5
+          args:
+           - --entryPoints.web.address=:4180
+           - --providers.file.directory=/traefik
+           - --log.level=DEBUG
+          ports:
+            - name: http
+              containerPort: 4180
+              protocol: TCP
+          volumeMounts:
+            - name: traefik-dynamic-config
+              mountPath: /traefik/traefik-dynamic-config.yaml
+              subPath: traefik-dynamic-config.yaml
+          resources:
+            requests:
+              cpu: 20m
+              memory: 16Mi
```

```suggestion
              memory: 32Mi
```
I did some testing and it was very easy for me to push the memory consumption above 16Mi and almost impossible (actually I didn't manage) to go above 32Mi. So I think with 32 we should be on the pretty save side (which is better than I expected).